carts-and-checkout/carts/cart-items/README.md

```diff
@@ -144,7 +156,15 @@ The `image` object will be empty if there is no main image assigned.
 | `meta.display_price.without_tax.value` | `object` | Tax exclusive totals for this cart item based on the quantity |
 | `meta.display_price.without_tax.value.amount` | `integer` | The raw total price this cart item line \(ex tax\) |
 | `meta.display_price.without_tax.value.currency` | `string` | The currency set for this cart item |
-| `meta.display_price.without_tax.value.formatted` | `string` | The tax inclusive formatted total of this cart item line based on the currency |
+| `meta.display_price.without_tax.value.formatted` | `string` | The tax inclusive formatted total of this cart item line based on the currency | 
```

Comment:
This should be unit, value is further down on line 167

Reply:
167 is for the `value` object in `tax`. That one pointed out is for `without_tax`.